versioned_docs/version-22.10/installation/installation-of-centreon-ha/installation-2-nodes.md

```diff
@@ -1529,9 +1530,9 @@ Ticket Constraints:
 ```bash
 Location Constraints:
 Ordering Constraints:
-  pcs constraint order stop centreon then demote ms_mysql-master
+  stop centreon then demote ms_mysql-master
 Colocation Constraints:
-  ms_mysql-master with centreon (score:INFINITY) (rsc-role:Master) (with-rsc-role:Started)
+  ms_mysql-clone with centreon (score:INFINITY) (rsc-role:Master) (with-rsc-role:Started)
```

Comment:
ms_mysql-**master** with centreon (score:INFINITY) (rsc-role:Master) (with-rsc-role:Started)

Reply:
It's ms_mysql-clone on RHEL8 but ms_mysql-master on CentOS7